go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.4.1
 	github.com/seccomp/containers-golang v0.3.2
-	github.com/sirupsen/logrus v1.4.2
+	github.com/sirupsen/logrus v1.5.0
```

Comment:
Can we avoid this update, there seems to be a deadlock: https://github.com/sirupsen/logrus/issues/1122

Reply:
Yup, done! Added a line under `replace` in go.mod to ensure it is at 1.4.2